packages/amplify-category-interactions/src/provider-utils/awscloudformation/auth-helper.js

```diff
@@ -1,5 +1,6 @@
 const os = require('os');
 const CategoryName = 'interactions';
+const { AmplifyError } = require('amplify-cli-core');
```

Comment:
```suggestion
const { AmplifyError } = require('@aws-amplify/amplify-cli-core');
```

make sure to pull latest dev. @evcodes was shuffling this stuff around.